scipy/stats/tests/test_stats.py

```diff
@@ -2229,7 +2333,7 @@ def test_ttest_rel():
         # check that nan in input array result in nan output
         anan = np.array([[1, np.nan], [-1, 1]])
         assert_equal(stats.ttest_rel(anan, np.zeros((2, 2))),
-                     ([0, np.nan], [1, np.nan]))
+                     (np.nan, np.nan))
```

Comment:
This hasn't been addressed. I think it is an appropriate change but it does change the expected output so it has potential to break some code. Do you think this is something we should consider at least mentioning somewhere? or is it ok to just go ahead with it?